From 511fd7a66c4535e95ee6192f9db2dabd7fdf0718 Mon Sep 17 00:00:00 2001
From: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Date: Fri, 2 Nov 2018 16:46:47 +0900
Subject: [PATCH 532/909] arm64: dts: r8a7795-es1-salvator-x: Add lvds node for
 du

commit bef75f5ca1aadbd0638775320de0003856b7559d from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds lvds endpoint node for du driver.

Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-es1-salvator-x.dts | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1-salvator-x.dts b/arch/arm64/boot/dts/renesas/r8a7795-es1-salvator-x.dts
index c1a56eab7b24..585c83e50494 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1-salvator-x.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1-salvator-x.dts
@@ -48,6 +48,16 @@
 		      "dclkin.0", "dclkin.1", "dclkin.2", "dclkin.3";
 };
 
+&lvds0 {
+	ports {
+		port@1 {
+			endpoint {
+				remote-endpoint = <&lvds_in>;
+			};
+		};
+	};
+};
+
 &ehci2 {
 	status = "okay";
 };
-- 
2.17.1

